docs/source/developers/continuous_integration/crossbow.rst

```diff
@@ -251,10 +249,9 @@ see its help page:
 .. _conda-forge packages: conda-recipes
 .. _Wheels: python-wheels
 .. _Linux packages: linux-packages
-.. _Create the queue repository: https://help.github.com/articles/creating-a-new-repository
+.. _Create the queue repository: https://docs.github.com/en/repositories/creating-and-managing-repositories/creating-a-new-repository
 .. _TravisCI: https://travis-ci.com/getting-started/
```

Comment:
```suggestion
.. _Travis-CI: https://travis-ci.com/getting-started/
```

Reply:
Please refer to the above discussion, I have replaced "TravisCI" to "Travis CI" in the entire document.